src/domain/events/__tests__/EventCard.test.tsx

```diff
@@ -190,3 +192,20 @@ it('renders multiday occurrence time correctly', async () => {
 
   expect(container).toMatchSnapshot();
 });
+
+it('renders date as tomorrow and ignores enrolment days', async () => {
+  Date.now = jest.fn(() => new Date(Date.UTC(2020, 8, 19)).valueOf());
```

Comment:
Could be that timer mocks would be a better fit here: https://jestjs.io/docs/timer-mocks

Reply:
jest-date-mock has been used in kultus

Reply:
I tried it with timer-mocks, but couldn't get it to work. I thought this is good enough.

Reply:
`advanceTo` was not working?

Reply:
`advanceTo` works! 👍  Changed the implementation to that. 

Had to look inside the code: https://github.com/hustcc/jest-date-mock/blob/master/src/mockDate.js. Sure it's easier to use, but I would say that it's more overwhelming and didn't give anything more here.